### ISSUE TITLE
Issue with convert when installing docs

### ISSUE THREAD

**Issue by shanosborne** [matplotlib/matplotlib#14756]
When building the docs with the `make html` command, if you already have a `convert` module somewhere else, you can get an error:

```
File "/Users/sosborne/miniconda3/envs/astroconda/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['convert', '--version']' returned non-zero exit status 1.
```

This should probably be fixed by checking if the `convert` module `matplotlib` is looking for is actually the correct module, and finding a work around if `convert` is from another package

**Comment by alexrudy** [comment]
Took a quick look at this – I think that [`_get_executable_info`](https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/__init__.py#L287) should probably be more forgiving, or raise a smaller subset of errors.

e.g. this error got raised via [`animation.ImageMagickBase`](https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/animation.py#L728), which tries to catch a `FileNotFoundError`, but `_get_executable_info` can raise other errors quite reasonably.

**Comment by timhoffm** [comment]
Not fully followed through, but it seems fixed by #14776. Please reopen if there is still something to do.
